### ISSUE TITLE
Questions about cameras in the data set

### ISSUE THREAD

**Issue by GHz12138** [AutonomousFieldRoboticsLab/SVIn#34]
Hello, thank you for your wonderful sharing. Recently, I extracted the original picture information from the rosbag format files of the cave data set and cemetery data set you shared before, and found that the occurrence frequency of the left and right images was unstable, resulting in a large difference in time stamps, and there was a pause in the middle, resulting in a different number of pictures from the left and right cameras. In addition, the original rosbag information had duplicate messages. For example, there are 13794 messages retrieved by rosabag info in the cave data set, but because there are actually 60 duplicate picture messages, I only 13734 pictures extracted. These problems may be the cause of frame loss and delay in the algorithm. Do you have any solutions?

/slave2/image_raw/cam_info      13794 msgs    : sensor_msgs/CameraInfo
/slave2/image_raw/compressed    13794 msgs : sensor_msgs/CompressedImage

**Comment by xxkklose** [comment]
@GHz12138 Hi, do you have the dataset they provided? Could you share a copy with me? Maybe I can help solve your problem.

**Comment by GHz12138** [comment]
> 嗨，你有他们提供的数据集吗？你能和我分享一份副本吗？也许我可以帮忙解决你的问题。

Yes, if I share the data set with you in accordance with the specification, and I don't know how to share it with you, is Baidu Netdisk OK?

**Comment by GHz12138** [comment]
> I don't know whether sharing the lab's data set infringes on his work, because his original data link seems to have failed. If it meets the standard requirements, I can share the data set with you. In addition, I don't know how to share it with you.

**Comment by GHz12138** [comment]
> 嗨，你有他们提供的数据集吗？你能和我分享一份副本吗？也许我可以帮忙解决你的问题。

I don't know whether sharing the lab's data set infringes on his work, because his original data link seems to have failed. If it meets the standard requirements, I can share the data set with you. In addition, I don't know how to share it with you.

**Comment by GHz12138** [comment]
@xxkklose 

I don't know whether sharing the lab's data set infringes on his work, because his original data link seems to have failed. If it meets the standard requirements, I can share the data set with you. In addition, I don't know how to share it with you, Baidu network disk?

**Comment by xxkklose** [comment]
@rekleitis Hi, since the link you provided is not working, could you please let our readers share your dataset internally for communication?
@GHz12138 Thanks, could you send Baidu Netdisk link or Google drive link to my mailbox klosexxk@gmail.com if allowed?

**Comment by yangyouze** [comment]
> @xxkklose
> 
> 我不知道分享实验室的数据集是否侵犯了他的工作，因为他原来的数据链接好像失效了。如果符合标准要求，我可以把数据集分享给你。另外，我不知道怎么分享给你，百度网盘？
你好，可不可以分享给我，我也是下载不了数据，您可不可以发送到1094960092@qq.com，或者是youzeyang44@gmail.com

**Comment by mzh0110** [comment]
> [@xxkklose](https://github.com/xxkklose)
> 
> I don't know whether sharing the lab's data set infringes on his work, because his original data link seems to have failed. If it meets the standard requirements, I can share the data set with you. In addition, I don't know how to share it with you, Baidu network disk?

你好，我也遇到了相同的问题，您可不可以发送到429582325@qq.com或者是maoziheng8186@gmail.com
